jax_unirep/evotuning.py

```diff
@@ -136,8 +159,29 @@ def length_batch_input_outputs(
     return xs, ys
 
 
+def get_batch_len(batched_seqs: Iterable[str]) -> Tuple[np.ndarray, List]:
+    """
+    Returns the average length of each batch as well as a full array of batch distribution.
+
+    :param batched_seqs: list of lists of sequences, grouped by length.
+    """
+    batch_lens = []
+    for batch in batched_seqs:
+        batch_lens.append(len(batch))
+
+    batch_lens = np.array(batch_lens)
+
+    return np.mean(batch_lens), batch_lens
```

Comment:
Maybe this would better fit in `utils.py`, where we also have the batching function itself.